src/providers/file/node.ts

```diff
@@ -9,21 +9,39 @@ async function node ( filePath = Utils.file.getActiveFilePath (), ...args ) {
 
   if ( !await Utils.file.is ( filePath ) ) return;
 
-  if ( !/\.js$/.test ( filePath ) ) return;
-
-  return {
-    provider: 'file.node',
-    configuration: {
-      name: 'Node',
-      type: 'node',
-      request: 'launch',
-      program: filePath,
-      args,
-      smartStep: true,
-      sourceMaps: true,
-      stopOnEntry: false
-    }
-  };
+  if ( /\.py$/.test ( filePath ) ) {
```

Comment:
I like exit conditions better, basically check if have to exit and return, rather than wrapping everything in a block (the less indentation/the flatter the file the better.

Reply:
Are you suggesting:add one more check? etc

  if (!/\.js$/.test ( filePath ) && !/\.py$/.test ( filePath ) ) return

  if ( /\.py$/.test ( filePath ) ) {

    return {
      provider: 'file.python',
      configuration: {
        name: 'Python',
        type: 'python',
        request: 'launch',
        program: filePath,
        args,
        smartStep: true,
        sourceMaps: true,
        stopOnEntry: false
      }
    };
  }

  if ( /\.js$/.test ( filePath ) ) {

    return {
      provider: 'file.node',
      configuration: {
        name: 'Node',
        type: 'node',
        request: 'launch',
        program: filePath,
        args,
        smartStep: true,
        sourceMaps: true,
        stopOnEntry: false
      }
    };
  }

Reply:
> Are you suggesting:add one more check? etc

No I mean in general instead of writing this:

```js
if ( something ) {
  do something
}
```

I prefer to write:

```js
if ( !something ) return;
do something
```

In order to have less indentation in the code.

Reply:
Previous code only have one case (.js) to check, so

if ( !something ) return;
do something

works. Now I need check two cases (.js and .py), so

if ( !somethingJS ) return;
do somethingJS

if ( !somethingPY ) return;
do somethingPY

does not work.

Do I miss anything there? Thanks.

Reply:
Correct, the other problem is that the file you modified handles only Node.js files, hence you can find it under `file/node.ts`.

You shouldn't have modified it, instead you should create a new file under `file/python.ts`

Reply:
Updated. Basically, I've matched all "node" with "python". 

However, because 1. the installed extension does not support python 2. the project does not have .vscode folder for debugging 3. I'm new to vscode ext, I am not able to create a .vscode folder to run/debug the change.

May you please take a look to see if it's OK.

Thanks.

Reply:
@9527KGG As explained in the "[Contributing](https://github.com/fabiospampinato/vscode-debug-launcher#contributing)" section of the readme this extension is debugged with itself. You just have to install it, open this repo, and run "Debug Launcher: Auto". But since you want to actually test the debugging capabilities you might have to properly install this extension first, basically run `vsce package` and then install the resulting `.vsix` file.

I don't feel I can test this as I'm not familiar with the Python ecosystem enough.

Reply:
Thanks for ur instructions. Updated. Now, I can debug .js and .py.

May you pls take a look again?